makefile

```diff
@@ -28,8 +31,9 @@ test: debug
 bin/dfmt-test: $(SRC)
 	$(DC) $(DMD_TEST_FLAGS) $^ -of$@
 
-bin/dfmt: $(SRC)
-	$(DC) $(DMD_FLAGS) $^ -of$@
+bin/dfmt: views/VERSION
+	$(DC) $(DMD_FLAGS) $(SRC) -of$@
```

Comment:
Better filter files: `$(DC) $(DMD_FLAGS) $(filter %.d,$^) -of$@`

Reply:
`$^` are the dependencies right ? in this case `views/VERSION`
that would not work

Reply:
`$^` are all the dependencies, yes, but `$(filter %.d,$^)` are only the dependencies matching `*.d` so to say, which is exactly what you want to pass to DMD, so it will work, it does for Makd :)

Reply:
this expands to
`dmd -O -inline -dip25 -w -Ilibdparse/src -Isrc -Jviews  -ofbin/dfmt`
since views/VERSION is the only dependency